src/seedu/addressbook/data/person/Address.java

```diff
@@ -8,25 +8,42 @@
  */
 public class Address {
 
-    public static final String EXAMPLE = "123, some street";
-    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses can be in any format";
-    public static final String ADDRESS_VALIDATION_REGEX = ".+";
+    public static final String EXAMPLE = "123, Clementi Ave 3, #12-34, 231534";
+    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses must be in BLOCK, STREET, UNIT, POSTAL_CODE format";
+    public static final String ADDRESS_VALIDATION_REGEX = ".+,.+,.+,.+";
+    public static final String SPLIT_TOKEN = ", ";
+
+    public final int BLOCK_INDEX=0;
+    public final int STREET_INDEX=1;
+    public final int UNIT_INDEX=2;
+    public final int POSTAL_CODE_INDEX=3;
```

Comment:
Good to see you avoiding 'magic numbers' :)

But try to keep the code format consistent, i.e. `public final int BLOCK_INDEX = 0` instead of `public final int BLOCK_INDEX=0`